pytorch_to_returnn/torch/_C.py

```diff
@@ -122,17 +123,44 @@ class Size(tuple):  # type: Tuple[SizeValue, ...]
 
 class SizeValue(int):
   """
-  We extend this, to store extra information, e.g. such that this reflects the batch-dim.
+  We extend this, to store extra information, e.g. corresponding RETURNN dim tags.
   """
-  is_batch_dim: bool = False
-  merged_dims: list[SizeValue] = []
```

Comment:
Maybe you anyway want to have this annotation?

Reply:
For `is_batch_dim`, this seems outdated. For `merged_dims`, it's already available in the definition of `__new__`. But if you want we can keep it.

Reply:
Does PyCharm correctly finds the definition of `merged_dims`? I know this is the case for `__init__` but I don't know about `__new__`. If so, then this is ok.

Reply:
Yes it does, so I'll leave it like this.